libmachine/crashreport/crash_report.go

```diff
@@ -76,6 +78,32 @@ func Send(err error, context string, driverName string, command string) error {
 	return bugsnag.Notify(err, metaData, bugsnag.SeverityError, bugsnag.Context{String: context}, bugsnag.ErrorClass{Name: fmt.Sprintf("%s/%s", driverName, command)})
 }
 
+// Send through http the crash report to bugsnag need a call to Configure(apiKey) before
+func Send(err error, context string, driverName string, command string) error {
+	return SendWithFile(err, context, driverName, command, "")
+}
+
+func addFile(path string, metaData *bugsnag.MetaData) {
+	if path == "" {
+		return
+	}
+	file, err := os.Open(path)
+	if err != nil {
+		log.Debug(err)
+		return
+	}
+	defer file.Close()
+	var buffer bytes.Buffer
```

Comment:
I think you probably just want `ioutil.ReadAll` instead of doing the scanning, right? https://golang.org/pkg/io/ioutil/#ReadAll

Then the code is just:

```go
data, err := ioutil.ReadAll(file)
if err != nil {
    log.Debug(err)
    return
}
metaData.Add("logfile", filepath.Base(path), string(data))
```